lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py

```diff
@@ -167,6 +166,76 @@
 '''
 
 RETURN = '''
+state:
+    description:
+        - Current state of the DNS record set.
+    returned: always
+    type: complex
+    contains:
+        id:
+            description:
+                - The DNS record set ID.
+            returned: always
+            type: str
+            sample: "/subscriptions/xxxx......xxx/resourceGroups/v-xisuRG/providers/Microsoft.Network/dnszones/b57dc95985712e4523282.com/A/www"
+        name:
+            description:
+                - Relate name of the record set.
+            returned: always
+            type: str
+            sample: 'www'
+        fqdn:
+            description:
+                - Fully qualified domain anme of the record set.
+            return: always
+            type: str
+            sample: www.b57dc95985712e4523282.com
+        etag:
+            description:
+                - The etag of the record set.
+            return: always
+            type: str
+            sample: 692c3e92-a618-46fc-aecd-8f888807cd6c
+        provisioning_state:
+            description:
+                - The DNS record set state.
+            return: always
+            type: str
+            sample: Succeeded
+        target_resource:
+            description:
+                - The target resource of the error message.
+            return: always
+            type: str
+            sample: contoso.com
+        ttl:
+            description:
+                - The TTL(time-to-live) of the records in the records set.
+            return: always
+            type: int
+            sample: 3600
+        type:
+            description:
+                - The type of DNS record in this record set.
+            return: always
+            type: str
+            sample: Microsoft.Network/dnsZones/A
+        arecords:
+            description:
+                - A list of records in the record set.
+            return: always
+            type: list
+            sample: [
+            {
+                "ipv4_address": "192.168.100.102"
```

Comment:
These examples should use the documentation-safe IPv4 range of 192.0.2.0 through 192.0.2.24. These look like they might be real IP addresses, which we should not include in documentation. 
```suggestion
                "ipv4_address": "192.0.2.0"
```